cpp/Marc21Tests.cc

```diff
@@ -0,0 +1,262 @@
+//
+// Created by quboo01 on 13.09.16.
+//
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "util.h"
+#include "File.h"
+#include "MarcUtil.h"
+#include "MarcWriter.h"
+#include "MarcRecord.h"
+#include "MarcReader.h"
+#include "MarcWriter.h"
+#include "WallClockTimer.h"
+
+void Usage() {
+    std::cerr << "usage: " << progname << " marc_input marc_output\n";
+
+    std::exit(EXIT_FAILURE);
+}
+
+static const size_t INSERTED_FIELDS_COUNT = 20;
```

Comment:
All top-level declarations should be separated by 2 blank lines from other top-level declarations.